.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+# 워크플로우의 이름
+name: CI - Build and Push Docker Image to Docker Hub
+
+# 워크플로우 트리거 -> main 브랜치에 푸쉬할 때
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)